mtenn/config.py

```diff
@@ -749,3 +748,127 @@ def _build(self, mtenn_params={}):
+    )
+    reduce_op: str = Field(
+        "sum", 
+        description="The type of reduction operation to apply. ['sum', 'mean']"
+    )
+    mean: float | None = Field(0.0, description="The mean of the output distribution.")
+    std: float | None = Field(1.0, description="The standard deviation of the output distribution.")
+    derivative: bool = Field(
+        False, 
+        description="Whether to compute the derivative of the output with respect to the positions."
+    )
+
+    @root_validator(pre=False)
+    def validate(cls, values):
+        # Make sure the grouped stuff is properly assigned
+        ModelConfigBase._check_grouped(values)
+
+        # Make sure atomref length is correct (this is required by PyG)
+        atomref = values["atomref"]
+        if (atomref is not None) and (len(atomref) != 100):
+            raise ValueError(f"atomref must be length 100 (got {len(atomref)})")
+
+        return values
+    
+    
+
+    def _build(self, mtenn_params={}):
+        """
+        Build an MTENN ViSNet Model from this config.
+
+        Parameters
+        ----------
+        mtenn_params: dict
+            Dict giving the MTENN Readout. This will be passed by the `build` method in
+            the abstract base class
+
+        Returns
+        -------
+        mtenn.model.Model
+            MTENN ViSNet Model/GroupedModel
+        """
+        # Create an MTENN ViSNet model from PyG ViSNet model
+
+        from mtenn.conversion_utils.visnet import HAS_VISNET
+        if HAS_VISNET:
+            from mtenn.conversion_utils import ViSNet
+
+            model = ViSNet(
+                lmax=self.lmax,
+                vecnorm_type=self.vecnorm_type,
+                trainable_vecnorm=self.trainable_vecnorm,
+                num_heads=self.num_heads,
+                num_layers=self.num_layers,
+                hidden_channels=self.hidden_channels,
+                num_rbf=self.num_rbf,
+                trainable_rbf=self.trainable_rbf,
+                max_z=self.max_z,
+                cutoff=self.cutoff,
+                max_num_neighbors=self.max_num_neighbors,
+                vertex=self.vertex,
+                reduce_op=self.reduce_op,
+                mean=self.mean,
+                std=self.std,
+                derivative=self.derivative,
+                atomref=self.atomref,
+            )
+            combination = mtenn_params.get("combination", None)
+            pred_readout = mtenn_params.get("pred_readout", None)
+            comb_readout = mtenn_params.get("comb_readout", None)
+
+            return ViSNet.get_model(
+                model=model,
+                grouped=self.grouped,
+                fix_device=True,
+                strategy=self.strategy,
+                combination=combination,
+                pred_readout=pred_readout,
+                comb_readout=comb_readout,
+            )
+
+        else:
+            raise ImportError("ViSNet not found. Is your PyG >=2.5.0?")
```

Comment:
Give more information and link to an issue.

```suggestion
            raise ImportError("ViSNet not found. Is your PyG >=2.5.0, you may need to use a PyG-nightly build, see issue #XXX")
```